DistributionPackages/Neos.Demo.UserRegistration/Configuration/Policy.yaml

```diff
@@ -1,19 +1,4 @@
 privilegeTargets:
-  'Neos\Media\Security\Authorization\Privilege\ReadAssetPrivilege':
-    'Neos.Demo:SeeOwnAssetCollectionContent':
-      matcher: 'isInCollection("context.userInformation.personalWorkspaceName")'
```

Comment:
Here we need to find another way as the userInformation.personalWorkspaceName does not exists anymore. We need to use it like in the AddUserWorkpaceAction.